pkg/virt-controller/services/net_binding.go

```diff
@@ -0,0 +1,30 @@
+package services
+
+import (
+	"fmt"
+
+	v1 "kubevirt.io/api/core/v1"
+
+	"kubevirt.io/kubevirt/pkg/hooks"
+)
+
+func BindingPluginSidecarList(vmi *v1.VirtualMachineInstance, config *v1.KubeVirtConfiguration) (hooks.HookSidecarList, error) {
```

Comment:
We need a context for this being network related. Perhaps just add `Net` as a prefix.